editor/code_editor.cpp

```diff
@@ -1543,17 +1543,10 @@ void CodeTextEditor::_update_text_editor_theme() {
 }
 
 void CodeTextEditor::_on_settings_change() {
```

Comment:
I'm pretty sure this was indirect to make sure the theme was loaded first, need to check that still works okay, or it should be safe to differ the emit of `load_theme_settings`. In any case, can merge these two methods now it's a direct call.

Reply:
OK whatever the reason doesn't seem to break anything or require the use of MessageQueue anymore. Swapping themes works fine, no issues.
<img width="877" alt="Screenshot 2021-08-15 at 02 35 59" src="https://user-images.githubusercontent.com/748770/129464211-531b9767-5196-4f9d-81f4-4107a6bfb920.png">